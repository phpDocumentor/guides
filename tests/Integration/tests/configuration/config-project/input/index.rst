==============
Document Title
==============

Lorem Ipsum Dolor.

title:
    |project|

version:
    |version|

release:
    |release|

copyright:
    |copyright|
