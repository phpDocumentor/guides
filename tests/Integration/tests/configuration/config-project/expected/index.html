<!-- content start -->
    <div class="section" id="document-title">
    <h1>Document Title</h1>

            <p>Lorem Ipsum Dolor.</p>
            <dl>
                        <dt>title:</dt>

        <dd>Title</dd>                        <dt>version:</dt>

        <dd>12.4</dd>                        <dt>release:</dt>

        <dd>12.4.9-dev</dd>                        <dt>copyright:</dt>

        <dd>since 1998 phpDocumentor Team and Contributors</dd>    </dl>

    </div>

<!-- content end -->
